feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -31,7 +31,14 @@ export class ConfiguratorPriceComponent {
     if (this.formula.priceTotal) {
       return this.priceTotal;
     } else {
-      return '+ ' + this.formula.price?.formattedValue;
+      if (
+        this.formula.price?.value !== undefined &&
+        this.formula.price?.value >= 0
+      ) {
+        return '+' + this.formula.price?.formattedValue;
+      } else {
+        return this.formula.price?.formattedValue || '';
```

Comment:
Plz, explain what you expected to get here. 
`this.formula.price?.formattedValue || ''` syntax is not known.
According to debugger `this.formula.price?.formattedValue` will be returned.